src/main/kotlin/no/nav/familie/ks/sak/kjerne/logg/LoggService.kt

```diff
@@ -82,4 +82,18 @@ class LoggService(
             )
         )
     }
+
+    fun opprettRegistrertSøknadLogg(behandlingId: Long, søknadGrunnlagFinnesFraFør: Boolean) {
```

Comment:
```suggestion
    fun opprettRegistrertSøknadLogg(behandlingId: Long, aktivSøknadGrunnlagFinnesFraFør: Boolean) {
```

Bare slik at man ikke bruker denne metoden og setter søknadGrunnlagFinnesFraFør til true hvis det finnes inaktive grunnlag 😛